book.common/src/main/java/com/happystays/book/common/events/BookingSuccessEvent.java

```diff
@@ -0,0 +1,22 @@
+package com.happystays.book.common.events;
+
+
+import com.happystays.book.common.dto.Pnr;
+import com.happystays.cqrs.core.events.BaseEvent;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+public class BookingSuccessEvent extends BaseEvent {
+
+    private List<Pnr> pnrList;
+    private List<String> users;
```

Comment:
what is private List<String> users? what is relation between pnrList and string of users. If this is meant for userId, do we need this here. Its already there is Pnr. So what is the point of giving user id here